.ci/monolithic-linux.sh

```diff
@@ -28,10 +27,14 @@ if [[ -n "${CLEAR_CACHE:-}" ]]; then
   ccache --clear
 fi
 
+mkdir -p artifacts/reproducers
```

Comment:
Note that this will not end up being uploaded on the Github side, although I should probably set something up to get that working.